packages/react-core/src/components/Page/PageBreadcrumb.tsx

```diff
@@ -39,10 +41,18 @@ export const PageBreadcrumb = ({
   hasShadowTop = false,
   hasShadowBottom = false,
   hasOverflowScroll = false,
+  'aria-label': ariaLabel,
   ...props
 }: PageBreadcrumbProps) => {
   const { height, getVerticalBreakpoint } = React.useContext(PageContext);
 
+  React.useEffect(() => {
+    if (hasOverflowScroll && !ariaLabel) {
+      /* eslint-disable no-console */
+      console.warn('An accessible aria-label is required when hasOverflowScroll is set to true.');
```

Comment:
I would add the component name to warnings to make it easier for consumer to identify what component needs attention.
e.g.  "PageBreadcrumb: An accessible aria-label is required when hasOverflowScroll is set to true.'